libbpf-tools/runqslower.bpf.c

```diff
@@ -88,8 +88,10 @@ int handle__sched_switch(u64 *ctx)
 		return 0;
 
 	event.pid = pid;
+	event.prev_pid = prev->pid;
 	event.delta_us = delta_us;
 	bpf_probe_read_kernel_str(&event.task, sizeof(event.task), next->comm);
+	bpf_probe_read_kernel_str(&event.prev_task, sizeof(event.task), prev->comm);
```

Comment:
Looks good overall, except this small nit.

Would be better to have `sizeof(event.prev_task)`.

Reply:
Fixed and force pushed!